Library/Formula/sqlmap.rb

```diff
@@ -1,10 +1,9 @@
 class Sqlmap < Formula
   desc "Penetration testing for SQL injection and database servers"
   homepage "http://sqlmap.org"
-  url "https://github.com/sqlmapproject/sqlmap/archive/0.9.tar.gz"
-  sha256 "77b2b0ac7844cbb045baa8d477ca11a9f3ca736b7e0b340787c8a181a2fc718f"
+  url "https://github.com/sqlmapproject/sqlmap/archive/1.0.tar.gz"
+  sha256 "6818c74d6bd5525879beedc9baaef404534cd5b8c0f671476367783fb3ef64a6"
   head "https://github.com/sqlmapproject/sqlmap.git"
```

Comment:
It’s only needed when building from `HEAD` using `--HEAD` which your patch removed support for.